source/kubernetes.rst

```diff
@@ -74,7 +74,14 @@ Install Percona XtraDB Cluster on Kubernetes
    More details about secrets can be found in a `separate
    section <../configure/users>`__.
 
-5. After the operator is started and user secrets are added, Percona
+5. Apply pre-generated certificates with the following command:
```

Comment:
user shouldn't use pre-generated certificates, it is highly unrecommended! we should add a link to TLS doc and explain that it is only for testing purposes he can use pre-generated certificates.